RecoEgamma/ElectronIdentification/python/idExternalRemapForPAT_cff.py

```diff
@@ -0,0 +1,57 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoEgamma.ElectronIdentification.idExternalRemap_cfi import *
+
+eidLooseGSFixed = eidLoose.clone(
+    candidateProducer = cms.InputTag('gedGsfElectronsGSFixed'),
+    newToOldObjectMap = cms.InputTag('gedGsfElectronsGSFixed'),
+)
+
+eidRobustHighEnergyGSFixed = eidRobustHighEnergy.clone(
+    candidateProducer = cms.InputTag('gedGsfElectronsGSFixed'),
+    newToOldObjectMap = cms.InputTag('gedGsfElectronsGSFixed'),
+)
+
+eidRobustLooseGSFixed = eidRobustLoose.clone(
+    candidateProducer = cms.InputTag('gedGsfElectronsGSFixed'),
+    newToOldObjectMap = cms.InputTag('gedGsfElectronsGSFixed'),
+)
+
+eidRobustTightGSFixed = eidRobustTight.clone(
+    candidateProducer = cms.InputTag('gedGsfElectronsGSFixed'),
+    newToOldObjectMap = cms.InputTag('gedGsfElectronsGSFixed'),
+)
+
+eidTightGSFixed = eidTight.clone(
+    candidateProducer = cms.InputTag('gedGsfElectronsGSFixed'),
+    newToOldObjectMap = cms.InputTag('gedGsfElectronsGSFixed'),
+)
+
+ElectronIDExternalProducerRemapSequence = cms.Sequence(
```

Comment:
it looks like a duplicate name also present in idExternalRemap_cfi
Better rename to avoid wrong one running.

Why is idExternalRemap_cfi file needed if it's used only to be loade in this file?